src/utils/payloadStringify.js

```diff
@@ -277,6 +305,7 @@ const shallowMaskByRegex = (payload, regexes) => {
 };
 
 export const shallowMask = (context, payload) => {
+  logger.debug('Shallow masking payload', { context, payload });
```

Comment:
Logging the entire payload might incur a high Cloudwatch cost. I think logging `payloadKeys: Object.keys(payload)` instead might be better without losing important data in this context

Reply:
Done